core/ui_mixins/quick_panel.py

```diff
@@ -157,6 +157,7 @@ class PaginatedPanel:
     limit = 6000
     selected_index = None
     on_highlight = None
+    is_this_selected = None
```

Comment:
Yes, I squashed into 514b317, Blame option: Pick a new commit, if you are on a commit, set it as selected.